### PR TITLE
IssueId #219476 fix: Next button is not visible in level 9 below screen resolution 1300

### PR DESCRIPTION
**Pull request by ajinkyapandetekdi**
## Summary by CodeRabbit

- **New Features**
  - Added a default function to handle language modal opening in the ProfileHeader component.
  
- **Improvements**
  - Updated message display logic in SpeakSentenceComponent based on iframe context.
  - Changed font size in WordsOrImage component to a responsive value.
  - Enhanced audio recording feedback handling in AudioCompare.js.
  - Improved completion message handling and screen disabling in Practice component.

### CODE REVIEW

File: src/components/Assesment/Assesment.jsx [hunk 1]
```diff
@@ -327,7 +327,21 @@ export const MessageDialog = ({
 };
 
 export const ProfileHeader = ({
-  setOpenLangModal,
+  setOpenLangModal = () => {
+    if(window.self !== window.top){
+      const alertMessage = "Go to homescreen to change language";
+      if (window?.parent) {
+        window.parent.postMessage({
```
Comment (gouravmore) [inline src/components/Assesment/Assesment.jsx:334]: why post message is required here?

File: src/components/DiscoverSentance/DiscoverSentance.jsx [hunk 2]
```diff
@@ -51,13 +51,21 @@ const SpeakSentenceComponent = () => {
...
+              icon:'success',
+              message: 'storylingo-alert',
+            });
+          }
+        }
+        else{
+          alert("You have successfully completed assessment " + assesmentCount);
+        }
+        setDisableScreen(false);
+      }, 3000);
```
Comment (gouravmore) [inline src/components/DiscoverSentance/DiscoverSentance.jsx:68]: 3 sec delay is not acceptable in application

File: src/utils/AudioCompare.js [hunk 3]
```diff
@@ -151,11 +151,19 @@ export default class AudioRecorderCompair extends Component {
           if (this.state.soundDetected) {
             this.props.setRecordedAudio(temp_audioSrc);
           } else {
-            // this.props.setRecordedAudio(temp_audioSrc);
-            this.props?.setOpenMessageDialog({
```
Comment (gouravmore) [inline src/utils/AudioCompare.js:155]: why this is removed?

**Comment by ajinkyapandetekdi** [comment]
Raised another PR for this from different branch

**Pull request closed without merging**